### PR TITLE
chore(*): Install babel

### PR DESCRIPTION
**Pull request by RSNara**
Just some minor changes:
- Installed a preset to transpile the features in ES6 not present in Node 6 (ie: the module syntax).
- `node-server.js` now imports the `app` and a function `startServer`. This is for testing purposes.

I didn't update any of the other code, because I don't know how the rest of the application will be affected. Lets leave it in place and only modify the code we need to.

### CODE REVIEW

**Comment by Bbowen100** [comment]
:shipit:

**Comment by mandishshah** [comment]
Did you guys just give yourself a :shipit:?

File: server/constants/.gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
-*
+secrets.js
```
Comment (mandishshah) [inline server/constants/.gitignore:1]: How dare you change this?

**Comment by mandishshah** [comment]
If you guys worked together then can we not have one give :shipit: to another? Defeats the purpose

**Comment by RSNara** [comment]
I didn't work together with brandon on this.

**Pull request merged**